### ISSUE TITLE
Numbers in the online marketing size tool do not reflect the new Kentico 10 limits

### ISSUE THREAD

**Issue by Craigath** [Kentico/KInspector#166]
Currently the limits set in the online marketing size tool are set to this:
•	OM_Activity > 10 000 000,
•	OM_Contact > 1 000 000,
•	OM_ContactGroup > 50,
•	OM_Rule > 50

But these numbers have changed (they can be a lot higher) in Kentico 10. Can this be changed?

**Comment by ChristopherJennings** [comment]
Good catch. The module needs to be updated to fork based on the version. The module should be updated to have configurable values for each and upon initialization, load the proper limits based on version. 
@Craigath, do you want to try to update this and submit a PR or should I mark this as up for grabs? Let me know if you need more details.

**Comment by Craigath** [comment]
Let’s mark it up for grabs, as I have little to no time the upcoming weeks ☺

Thanks for the invite though!

Regards,
Jos
